wemake_python_styleguide/visitors/ast/statements.py

```diff
@@ -457,3 +460,38 @@ def _check_augmented_assign_pattern(
 
         if name_nodes.is_same_variable(node.targets[0], node.value.left):
             self.add_violation(AugmentedAssignPatternViolation(node))
+
+
+@final
+class WrongMethodParametersVisitor(BaseNodeVisitor):
+    """Ensures that all parameters follow our rules."""
+
+    _no_tuples_collections: ClassVar[AnyNodes] = (
+        ast.List,
+        ast.Set,
+    )
+
+    def visit_Call(self, node: ast.Call) -> None:
+        """Checks call arguments indentation."""
+        all_args = call_args.get_all_args(node)
```

Comment:
Let's move this line into `_check_tuple_parameters_types`

Reply:
sorry, don't understand